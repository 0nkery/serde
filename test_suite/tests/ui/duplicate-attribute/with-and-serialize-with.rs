#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
struct S {
    #[serde(with = "w", serialize_with = "s")]
    x: (),
}

fn main() {}
