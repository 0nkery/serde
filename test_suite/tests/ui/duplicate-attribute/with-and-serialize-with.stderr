error: duplicate serde attribute `serialize_with`
 --> $DIR/with-and-serialize-with.rs:6:25
  |
6 |     #[serde(with = "w", serialize_with = "s")]
  |                         ^^^^^^^^^^^^^^

error: aborting due to previous error

